src/__test__/redux/reducers/cellInfoReducer.test.js

```diff
@@ -4,30 +4,6 @@ import {
   CELL_INFO_UPDATE, CELL_INFO_FOCUS, CELL_INFO_UNFOCUS,
 } from '../../../redux/actionTypes/cellInfo';
```

Comment:
While you're editing this file, can you please change these references to use absolute path?